frame/babe/src/lib.rs

```diff
@@ -357,12 +357,12 @@ pub mod pallet {
 					);
 				}
 
-				if let Some(vrf_signature) = pre_digest.vrf_signature() {
+				if let Some(signature) = pre_digest.vrf_signature() {
 					let randomness: Option<BabeRandomness> = Authorities::<T>::get()
 						.get(authority_index as usize)
 						.and_then(|(authority, _)| {
 							let public = authority.as_inner_ref();
-							let transcript = sp_consensus_babe::make_transcript(
+							let transcript = sp_consensus_babe::make_vrf_transcript(
```

Comment:
The same here: can use `make_vrf_sign_data` here, instead of converting `into_sign_data` below?

Reply:
No because here we just need the transcript to generate random bytes. The signing data is required only in the `debug_assert` where we optionally check the corresponding signing data associated to the transcript

Reply:
I missed that, thanks.